pom.xml

```diff
@@ -228,6 +229,23 @@
         <version>2.6.2</version>
         <scope>compile</scope>
       </dependency>
+
+      <!-- JAXB API and implementation needed for Java 9 -->
+      <dependency>
+        <groupId>javax.xml.bind</groupId>
+        <artifactId>jaxb-api</artifactId>
+        <version>2.3.0</version>
+      </dependency>
+      <dependency>
+        <groupId>org.glassfish.jaxb</groupId>
+        <artifactId>jaxb-runtime</artifactId>
+        <version>2.3.0</version>
+      </dependency>
+      <dependency>
```

Comment:
While I can understand easily the runtime I am unsure of why javax.activation is required. Comment ?

Reply:
jaxb depends on `javax.activation.DataSource` , I don't know why this is not transitive dependency but it will fail when not provided.